src/codegen/codegen_c_visitor.cpp

```diff
@@ -3393,14 +3393,14 @@ void CodegenCVisitor::print_net_receive_common_code(Block* node, bool need_mech_
         printer->add_newline();
         for (auto& parameter: parameters) {
             auto name = parameter->get_node_name();
-            VarUsageVisitor vu;
-            auto var_used = vu.variable_used(node, "(*" + name + ")");
+            bool var_used = VarUsageVisitor().variable_used(node, "(*" + name + ")");
```

Comment:
```suggestion
            bool var_used = VarUsageVisitor().variable_used(node, "*" + name);
```
I don't think `()` is needed.